Tools/DumpRenderTree/mac/Configurations/DumpRenderTreeLibrary.xcconfig

```diff
@@ -28,10 +28,6 @@
 PRODUCT_NAME = DumpRenderTree;
 SKIP_INSTALL = YES;
 
-// Workaround for "WebKit is not available..." error (rdar://91668054)
-DISABLE_SDK_METADATA_PARSING[sdk=appletv*] = YES;
-DISABLE_SDK_METADATA_PARSING[sdk=watch*] = YES;
-
```

Comment:
Confirmed that while this would break builds for tv/watchOS on Xcode 13.x, we only build embedded WebKit with 14.x.